sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/FileStreamOptions.scala

```diff
@@ -53,4 +53,15 @@ class FileStreamOptions(parameters: CaseInsensitiveMap) extends Logging {
   /** Options as specified by the user, in a case-insensitive map, without "path" set. */
   val optionMapWithoutPath: Map[String, String] =
     parameters.filterKeys(_ != "path")
+
+  /** Whether to scan new files first. */
```

Comment:
Can you elaborate this comment further. In a trigger when it finds unprocessed files, it will first process the latest file.

Also, isnt latest more common than newest?